android/app/src/main/java/labs/pooh/mycanteen/util/ImageUtils.kt

```diff
@@ -0,0 +1,30 @@
+package labs.pooh.mycanteen.util
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import java.lang.Exception
+import java.net.HttpURLConnection
+import java.net.URL
+
+fun downloadImageFrom(url: URL): Bitmap? {
+    return try {
+        val connection = url.openConnection() as HttpURLConnection
+        connection.doInput = true
+        connection.connect()
+        val inputStream = connection.inputStream
+        BitmapFactory.decodeStream(inputStream)
+    }
+    catch (e: Exception) {
+        null
```

Comment:
Do we handle these exceptions? Even if so, I would still log debug to logcat.

Reply:
as the exception result we get null instead of object and we have to handle this problem (we deal with Bitmap? type)